docker.Makefile

```diff
@@ -41,84 +41,70 @@ build_e2e_image:
 	docker build -t $(E2E_IMAGE_NAME) --build-arg VERSION=$(VERSION) --build-arg GITCOMMIT=$(GITCOMMIT) -f ./dockerfiles/Dockerfile.e2e .
 
 
-# build executable using a container
-binary: build_binary_native_image
+binary: build_binary_native_image ## build the CLI
 	docker run --rm $(ENVVARS) $(MOUNTS) $(BINARY_NATIVE_IMAGE_NAME)
 
-build: binary
+build: binary ## alias for binary
 
-
-# clean build artifacts using a container
 .PHONY: clean
-clean: build_docker_image
+clean: build_docker_image ## clean build artifacts
 	docker run --rm $(ENVVARS) $(MOUNTS) $(DEV_DOCKER_IMAGE_NAME) make clean
 
-# run go test
 .PHONY: test-unit
-test-unit: build_docker_image
+test-unit: build_docker_image # run unit tests (using go test)
 	docker run --rm $(ENVVARS) $(MOUNTS) $(DEV_DOCKER_IMAGE_NAME) make test-unit
 
-.PHONY: test
+.PHONY: test ## run unit and e2e tests
 test: test-unit test-e2e
 
-# build the CLI for multiple architectures using a container
 .PHONY: cross
-cross: build_cross_image
+cross: build_cross_image ## build the CLI for macOS and Windows
 	docker run --rm $(ENVVARS) $(MOUNTS) $(CROSS_IMAGE_NAME) make cross
 
 .PHONY: binary-windows
-binary-windows: build_cross_image
+binary-windows: build_cross_image ## build the CLI for Windows
 	docker run --rm $(ENVVARS) $(MOUNTS) $(CROSS_IMAGE_NAME) make $@
 
 .PHONY: binary-osx
-binary-osx: build_cross_image
+binary-osx: build_cross_image ## build the CLI for macOS
 	docker run --rm $(ENVVARS) $(MOUNTS) $(CROSS_IMAGE_NAME) make $@
 
-.PHONY: watch
```

Comment:
Watch should also be removed in the Makefile no?

Reply:
Yea, thanks for pointing it out.